uint8 iskeepCurSpeed = 1
uint8 notkeepCurSpeed = 0

float64 EgoSpeed
float64 PreSpeed
float64 LimitSpeed
float64 Dis2Stopline

uint32 intersectionID

float64 Latitude
float64 Longitude
float64 Height


float64 upperSpeed
float64 bottomSpeed
uint8 keepCurSpeed

uint8 Str_is_val
float64 StrColor
float64 StrlikelyEndTime

uint8 Left_is_val
float64 LeftColor
float64 LeftlikelyEndTime

uint8 Right_is_val
uint8 RightColor
float64 RightlikelyEndTime
